tox.ini

```diff
@@ -34,27 +34,21 @@ whitelist_externals = open
 deps =
     tox
 commands =
-    tox -e py27-integration
-    tox -e py36-integration
-    tox -e pypy-integration
+    tox -e py27-requests-integration -- {posargs:-vvs}
+    tox -e py36-requests-integration -- {posargs:-vvs}
+    tox -e pypy-requests-integration -- {posargs:-vvs}
 
-[testenv:pypy-integration]
-deps =
-    tox
+[testenv:py27-requests-integration]
```

Comment:
probably better to leave these named as-is - easier to remember/type and afaict theres nothing requests specific about these integration tests.

Reply:
👍